Packs/Legacy/Integrations/EWSv2/EWSv2.py

```diff
@@ -946,6 +946,19 @@ def str_to_camel_case(snake_str):
     return str_to_camel_case(value)
 
 
+def email_ec(item):
+    return {
+        'CC': [] if not item.cc_recipients else [mailbox.email_address for mailbox in item.cc_recipients],
+        'BCC': [] if not item.bcc_recipients else [mailbox.email_address for mailbox in item.bcc_recipients],
+        'To': [] if not item.to_recipients else [mailbox.email_address for mailbox in item.to_recipients],
```

Comment:
1) are we sure we want an empty list as a default value and not omitting the key from the object?
2) when empty, what `item.*_ recipients` contain? empty list or None? if list then the condition is redundant.

Reply:
Changed to None